src/sentry/auth/superuser.py

```diff
@@ -41,8 +43,11 @@
 # the maximum time the session can stay alive
 MAX_AGE = getattr(settings, "SUPERUSER_MAX_AGE", timedelta(hours=4))
 
+# the maximum time the session can stay alive when accessing different orgs
+MAX_AGE_CHANGED_ORG = getattr(settings, "SUPERUSER_ORG_CHANGE_MAX_AGE", timedelta(minutes=15))
```

Comment:
is "changed org" the right name here? what do we think about something like "PRIVILEGED_ORG_ACCESS" instead?